.github/workflows/test.yml

```diff
@@ -13,7 +13,57 @@ jobs:
       - uses: denoland/setup-deno@v1
         with:
           deno-version: v1.x
-      - run: KAKAROT_ADDRESS=0x1 deno test --allow-env
+      # Checkout target branch and run tests
+      - name: Checkout target branch
+        run: |
+          git fetch origin ${{ github.head_ref }}
+          git checkout ${{ github.head_ref }}
```

Comment:
could we remove this and the following one using the `checkout` action or was that not possible?

Reply:
fixed, thanks